pkg/api/service/util.go

```diff
@@ -104,45 +87,16 @@ func NeedsHealthCheck(service *api.Service) bool {
 
 // GetServiceHealthCheckNodePort Return health check node port for service, if one exists
 func GetServiceHealthCheckNodePort(service *api.Service) int32 {
```

Comment:
Do you think it worth to get rid of this function? This is essentially checking `service.Spec.HealthCheckNodePort`. As the beta API is removed we don't need this abstraction layer anymore.